app.js

```diff
@@ -1,25 +1,26 @@
-const express = require('express')
-const logger = require('morgan')
-const cors = require('cors')
+const express = require("express")
+const morgan = require("morgan")
+const cors = require("cors")
 
-const contactsRouter = require('./routes/api/contacts')
+const contactsRouter = require("./routes/contacts");
+const router = require("./routes/contacts") 
+const app = express();
 
-const app = express()
+const formatsLogger = app.get("env") === "development" ? "dev" : "short";
 
-const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short'
-
-app.use(logger(formatsLogger))
+app.use(morgan(formatsLogger))
 app.use(cors())
 app.use(express.json())
+app.use(router);
```

Comment:
Aktualnie masz duplikat tych routerów

```
app.use(router);
app.use('/api/contacts', contactsRouter)
```
Jak dasz zapytanie `GET localhost:3000` to zwróci ci to samo co `GET localhost:3000/api/contacts` .Raczej powinno zostać tylko to z api/contacts